template/_package.json

```diff
@@ -111,6 +114,11 @@
     <%_ if (server !== 'none') { _%>
     "nodemon": "^1.18.9",
     <%_ } _%>
+    <%_ if (typescript) { _%>
+    "@nuxt/typescript-build": "^0.1.10",
+    "@types/node": "^10.14.16",
```

Comment:
Why are `@types/node` and `@types/webpack-env` needed ?

Reply:
this is my fault :( It need to be deleted

Thanks 👍